### DISCUSSION TITLE
PAT Security?

### DISCUSSION THREAD

**Discussion by Richienb** [community/community#24425] [Apps, API and Webhooks]
<p>If I created a public access token with no scopes selected, would it be safe to store it as plain text?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/richienb">@richienb</a>,</p>
<p>Thanks for being here! No scope would grant read-only access to public information (includes public user profile info, public repository info, and gists). While it’s certainly safer than a PAT with a scope, I personally wouldn’t recommend storing *any* tokens in plain text. It just helps to prevent errors and such.</p>
<p>Cheers!</p>
